extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -352,4 +357,20 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     }
     return true;
   }
+
+  addDefaultFallback() {
```

Comment:
Is there any reasonable way that this functionality could be moved into `amp-ad-api-handler`?  A4A needs the same behavior, and it would be best to share the code if possible.